insights/v1alpha1/types_insights.go

```diff
@@ -171,7 +171,7 @@ type GathererStatus struct {
 	// lastGatherDuration represents the time spent gathering.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Type=string
-	// +kubebuilder:validation:Pattern="^([1-9][0-9]*(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
+	// +kubebuilder:validation:Pattern="^(0|([0-9]+(\\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+)$s"
```

Comment:
s after the dollar does not look correct, and why do we now allow leading zeroes? Is that preferable?

Reply:
ahh good catch with the `s` character. 

wrt leading zeroes...hmm I am not sure. Hopefully it's not needed. I was surprised by the `2m0s` value so not sure if there can be something like `0m46s`. Probably not right

Reply:
It seems valid in Go to have duration like `0h54m` - https://play.golang.com/p/ahB1cojf5Rl